src/Form/index.tsx

```diff
@@ -1070,7 +1079,16 @@ function Form({
         b.properties.actions.some((action: any) => action.type === ACTION_NEXT)
       );
       const terminalStep = !hasNext && nextStep.next_conditions.length === 0;
-      if (terminalStep) eventData.completed = true;
+      if (terminalStep) {
+        const authIntegration = getAuthIntegrationMetadata(integrations);
+        // Completed should be false if it is a 'terminal step' in the middle of
+        // a login flow with steps configured for after login
+        const isTerminalStepAuth =
```

Comment:
wonder if all this logic should be a auth helper function in a utils file in the  auth directory

Reply:
realized we may not have an auth directory now, but could be nice to make a top level folder called auth, then put the LoginProvider, useAuth (maybe renamed to useFormAuth), and any utility functions in it